src/streamlink/plugins/vk.py

```diff
@@ -0,0 +1,86 @@
+import json
+import re
+import urllib.parse
```

Comment:
Make sure you use the `streamlink.compat` module to get the `url*` methods so that it works in Python2/3 :)